contracts/escrow/LemonadeEscrowFactoryV1.sol

```diff
@@ -25,6 +33,22 @@ contract LemonadeEscrowFactoryV1 is Ownable, ILemonadeEscrowFactory {
         return _signer;
     }
 
+    function setFeeCollector(address feeCollector) public onlyOwner {
```

Comment:
These functions are not properly ordered, see https://docs.soliditylang.org/en/latest/style-guide.html#order-of-functions

Reply:
reordered